### DISCUSSION TITLE
i18n redirect fallback for Google Bots

### DISCUSSION THREAD

**Discussion by Lion7de** [nuxt-modules/i18n#1739] [Q&A]
Hello!
I hope that i am in the right place for my issue.

My Problem:
My website has the main locale DE on the root page. It is currently only translated into EN available at the subpath /en. I would like to use the detectBrowserLanguage feature to redirect users who come from non-german speaking countries fallback to the english version of the website.

So my nuxt.config.js / i18n configuration looks like this:
```

i18n: {
    baseUrl: 'https://www.domain.com',
    strategy: 'prefix_except_default',
    lazy: true,
    locales: [
      {
        code: 'de',
        name: 'Deutsch',
        iso: 'de',
        file: 'de-de.json',
      },
      {
        code: 'en',
        name: 'English',
        iso: 'en',
        file: 'en-eu.json',
      },
    ],
    langDir: 'i18n/',
    defaultLocale: 'de',
    detectBrowserLanguage: {
      useCookie: true,
      cookieKey: 'i18n_redirected',
      redirectOn: 'root',
      fallbackLocale: 'en',
    },
    vueI18n: {
      fallbackLocale: 'de',
    },
  },
```

After that change, it seems that the Google Bots were always redirected to the english version of the page. Looking at the tested website for the root Page in the Google Search Console, it shows the english version of the main page. After that change, the count and ranking on keywords in Germany clearly dropped and English keywords raised.

So my question is: Is there any way to redirect users to the fallback locale /en but still make sure that Google can see the correct main page in german ? Did i do something wrong with the detectBrowserLanguage Config?

Nuxt Version: 2.15.18
i18n Version: 7.2.2

**Comment by TorbenHammes** [comment]
I have the same problem. Can somebody help? Is there already a solution?
